src/civetweb.c

```diff
@@ -6079,6 +6090,10 @@ static char *addenv(struct cgi_env_block *block, const char *fmt, ...)
 	char *added;
 	va_list ap;
 
+	if (env == NULL) {
+		return;
+	}
+
 	if (block == NULL || fmt == NULL) {
```

Comment:
This line (`if (block == NULL || fmt == NULL) {`) does not exist anymore in the current source.